deploy/roles/web/handlers/main.yaml

```diff
@@ -1,11 +1,18 @@
 ---
 
+# gzip files for nginx's gzip_static
+- name: gzip files
+  command: "{{ item }}"
+  loop:
+    - "find /var/www/playreading/public -name '*.gz' -exec rm {} ;"
+    - "find /var/www/playreading/public -type f ( -name '*.css' -o -name '*.js' -o -name '*.html' ) -exec gzip --best --keep {} ;"
+
+- name: fix selinux permissions
+  command: restorecon -R /var/www/playreading
+
 - name: restart application
   systemd:
     name: playreading.service
     daemon_reload: yes
     state: restarted
     enabled: yes
-
-- name: fix selinux permissions
-  command: restorecon -R /var/www/playreading
```

Comment:
This was moved up to fix the ordering since we want to fixup the permissions before restarting the application which uses those files